doc/visual-programming/source/widgets/data/csvfileimport.md

```diff
@@ -0,0 +1,63 @@
+CSV File Import
+===============
+
+Import a data table from a CSV formatted file.
+
+**Outputs**
+
+- Data: dataset from the .csv file
+- Data Frame: pandas DataFrame object
+
+The **CSV File Import** widget reads comma-separated files and sends the dataset to its output channel. File separators can be commas, semicolons, spaces, tabs or manually-defined delimiters. The history of most recently opened files is maintained in the widget.
+
+*Data Frame* output can be used in the [Python Script](../data/pythonscript.md) widget by connecting it to the `in_object` input (e.g. `df = in_object`). Then it can be used a regular DataFrame.
+
+### Import Options
+
+The import window where the user sets the import parameters. Can be re-opened by pressing *Import Options* in the widget.
+
+![](images/CSVFileImport-ImportOptions-stamped.png)
+
+1. File encoding. Default is UTF-8. See Encoding subchapter for details.
+2. Import settings:
+   - *Cell delimiter*:
+      - Tab
+      - Comma
+      - Semicolon
+      - Space
+      - Other (set the delimiter in the field to the right)
+   - *Quote character*: either " or '. Defines what is considered a text.
+   - *Number separators*:
+      - Grouping: delimiters for thousands, e.g. 1,000
+      - Decimal: delimiters for decimals, e.g. 1.234
+3. Column type: select the column in the preview and set its type.
+   - *Auto*: Orange will automatically try to determine column type. (default)
+   - *Numeric*: for continuous data types, e.g. (1.23, 1.32, 1.42, 1.32)
+   - *Categorical*: for discrete data types, e.g. (brown, green, blue)
+   - *Text*: for string data types, e.g. (John, Olivia, Mike, Jane)
+   - *Datetime*: for time variables, e.g. (1970-01-01)
+   - *Ignore*: do not output the column.
```

Comment:
Should also add a:

4.) Right click on the row header to the left to mark a row as a header, skipped or a normal data row.

(This needs to be made clearer from the GUI also)

Reply:
Will do.